### ISSUE TITLE
Unable to open some datasets from ECMWF/Copernicus

### ISSUE THREAD

**Issue by PeterPetrik** [lutraconsulting/MDAL#43]
see 
https://lists.osgeo.org/pipermail/qgis-developer/2018-November/055205.html

**Comment by saberraz** [comment]
The problem is that grib/netcdf formats compatible with gdal are recognised by mdal at the moment. 
The grib in the above link is attached and gdalinfo gives me error.


[global-waves-2018-08-01-0600-1800.grib.zip](https://github.com/lutraconsulting/MDAL/files/2560972/global-waves-2018-08-01-0600-1800.grib.zip)

**Comment by PeterPetrik** [comment]
@saberraz is this still the issue?

**Comment by saberraz** [comment]
Yes, it is still a problem. Note this is not a GDAL supported grib.
